backend/controllers/Post.controller.ts

```diff
@@ -0,0 +1,76 @@
+import { Post } from "../models/Post.model";
+import { Request, Response } from "express";
+import { Genre } from "../models/Genre.model";
+import { users } from "@clerk/clerk-sdk-node";
+
+interface PublicMetadata {
+  posts: any[]; // Update the type of posts as needed
+  "fav-genres": string[]; // Update the type of fav-genres as needed
+}
+
+//get all posts
+export const getAllPosts = async (req: Request, res: Response) => {
+  try {
+    const posts = await Post.find();
```

Comment:
`getallposts()` will be useful when we go through the discover page, since posts will be from different people. I'll add another method of posts made by a specific person (is this is what you mean) 
but for the post obj in user, I only did put the post.id as discussed earlier, as you can see in the clerk dashboard in the picture below
![image](https://github.com/chingu-voyages/v44-tier3-team-33/assets/65737942/edd725ae-3a9c-49ea-b62b-9d793017b90b)